index.js

```diff
@@ -1 +1,82 @@
 // implement your API here
+const express = require('express');
+const db = require('./data/db')
+
+const server = express();
+
+server.use(express.json());
+
+server.post('/users',(req, res) => {
+    const {name , bio} = req.body;
+
+    db.insert(name && bio)
+    .then(user => {
+        if(user !== user.name && user.bio) {
+            res.status(400).json({errorMessage: "Please provide name and bio for the user."})
```

Comment:
are you trying to say if user does not equal user.name and user.bio exists? If so, that's fine. If it's the error handling I am thinking of though, you are just instituting a check to see if name and bio exists, yes? 

do the following: 

```
if(!user.name || !user.bio) {
your logic here...
}
```